store/tikv/delete_range.go

```diff
@@ -36,6 +36,7 @@ type DeleteRangeTask struct {
 }
 
 // NewDeleteRangeTask creates a DeleteRangeTask. Deleting will not be performed right away.
+// WARMING: Currently this API may last some dirty keys in TiKV. Be careful using this API.
```

Comment:
Currently, this API may leave some waste key-value pairs uncleaned in TiKV. Be careful while using it.

Reply:
`WARNING`.